src/interactive/accordionPanelGroup.test.jsx

```diff
@@ -58,11 +58,14 @@ describe('AccordionPanelGroup', () => {
 		/>,
 	];
 
-	describe('isPrimitve', () => {
-		it('returns true when passed a primitive value', () => {
-			const values = [1, 'string', true, () => {}, [], {}];
-			const valueTypes = values.map(x => isPrimitive(x));
-			expect(valueTypes).toEqual([true, true, true, false, false, false]);
+	describe('getNewPanelState', () => {
+		it('returns expected new panel state list when passed accordionPanelGroup data', () => {
```

Comment:
would be nice to also have a test that asserts only the clicked panel index changes when multiselect is `false`